torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9221,6 +9234,99 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
     OpInfo('scatter',
            dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
            sample_inputs_func=sample_inputs_scatter,),
+    OpInfo('Tensor.bfloat16',
+           dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
+           supports_out=False,
+           sample_inputs_func=sample_inputs_conversion,
+           # The autograd test runner cannot handle functions that change dtype
+           supports_autograd=False,
+           skips=(
+               # This method not supported by TorchScript
+               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
```

Comment:
Given that this test is skipped by a lot of `OpInfo`'s we probably don't need a comment here. On the other hand, I would make the skip more precise by adding the skipped dtypes:
```suggestion
               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit', dtypes=(torch.float32, torch.complex64)),
```

This comment applies to all occurrences.

Reply:
Removed the comment. I didn't want to make the skip more precise: the reasoning is, `test_variant_consistency_jit` only seems to be run on some dtypes. However, no matter the dtype, `test_variant_consistency_jit` should fail the test because TorchScript simply doesn't support the operator. In the future, if someone comes along and adds more dtypes to `test_variant_consistency_jit`, then they would need to update the dtypes if we made it more precise.